### PR TITLE
python3: update to 3.7.2.

### PR DESCRIPTION
**Pull request by Kistelini**
I generated the list for rebuilds with
```
grep -r --include=template python3-devel | cut -d/  -f 2 >> rebuild
grep -r --include=template build_style=python-module | cut -d/  -f 2 >> rebuild
grep -r --include=template build_style=python3-module | cut -d/  -f 2 >> rebuild
xrevshlib libshiboken-python3 >> rebuild # remove freecad, python2 only
xrevshlib python3 >> rebuild
for p in $(xlocate /usr/lib/python3 | cut -d\	 -f 1 | sort -u | rev | cut -d- -f 2- | rev); do xsubpkg -m $p; done >> rebuild
sort -u -o rebuild rebuild
```
https://gist.github.com/Kistelini/3eeffc97743128b44569d8d9095e5e3a

I built all packages with `xbulk` but some are still causing trouble, ~~especially `python-PyQt5` is broken~~ `python-PyQt5` builds with the new `qt5-webkit` version.

These packages failed the `xbulk` but build after building them a second time (links to first build logs):
- `gufw` https://gist.github.com/Kistelini/ede3b05f903f9de32270c69837e33cd7
- `lightdm-gtk-greeter-settings` https://gist.github.com/Kistelini/6aaf7e77bff23ec64aaf83f1bf70d8e0
- `menulibre` https://gist.github.com/Kistelini/90e9b17596f731df164d5ab07502ca2c
- `mugshot` https://gist.github.com/Kistelini/3fc3df7e196e3327dcdd45a83b109546
- `udiskie` https://gist.github.com/Kistelini/2a9304aa7cca58c986fe1d62b020cbb3

These packages don't build at all:

- `pymol` https://gist.github.com/Kistelini/13b0cc7c7451ef52683b1e23b9b1a26d, could be fixed in master https://github.com/schrodinger/pymol-open-source/issues/17 but we don't package that right?
- `python-pytest-relaxed` is marked as broken, should it be removed?
- `scudcloud` needs at least a patch but upstream is dead anyways https://github.com/raelgc/scudcloud/issues/628, should it be removed?
- `variety` https://gist.github.com/Kistelini/92cf8bd29d76485dba392f4d12b57551

### CODE REVIEW

**Comment by maxice8** [comment]
Please add [ci skip] to the commits so it doesn't clog up the builders.

There is no way this can pass all CI without breaking 50 minutes limit.

**Comment by Kistelini** [comment]
Yeah I added it to the first commit, thought that was enough. Do I have to add it to all single commits?

**Comment by maxice8** [comment]
On Mon, Jan 14, 2019 at 06:08:47PM -0800, Kistelini wrote:
> Yeah I added it to the first commit, thought that was enough. Do I have to add it to all single commits?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/7142#issuecomment-454241040

all commits

File: srcpkgs/python-PyOpenGL-accelerate/template [hunk 1]
```diff
@@ -17,12 +17,3 @@ checksum=927f4670b893d46e2f6273ae938bf0a1db27ffae3336eba94813ccef6260c410
 post_install() {
 	vlicense license.txt LICENSE
 }
-
-python3-PyOpenGL-accelerate_package() {
```
Comment (Johnnynator) [inline srcpkgs/python-PyOpenGL-accelerate/template]: https://github.com/void-linux/void-packages/blob/master/Manual.md#pkg_remove

**Pull request closed without merging**